### DISCUSSION TITLE
Is there a way to ignore a header/param/query?

### DISCUSSION THREAD

**Discussion by josh-burton** [asteasolutions/zod-to-openapi#114] [Q&A]
I have a bunch of headers defined, but I'd like some to be excluded from the final spec.

Is there a way to do this?

**Comment by AGalabov** [comment]
Hello @josh-burton can you provide some additional information. 

Overall the short answer is "not from the library" but you can achieve such behavior from your own code (using JS/TS).

Some of my questions are:
1. Are those headers something that you do not want to add to the spec at all? I.e for none of your routes. If that is the case you can just not register them.
2. If they are being used for specific routes - then as long as you provide them for the appropriate routes only, then they would only be present in their respective documentation

Am I missing something?

**Comment by josh-burton** [comment]
@AGalabov I have a bunch of schemas that are used within my service, that I'm also registering against the registry.

Some of the fields in those schemas I really only want to use internally, and just want to skip/ignore them completely from the spec.

Maybe a param in the openapi extension called 'ignore' or something could work?
